docs/index.rst

```diff
@@ -0,0 +1,46 @@
+..
+    Copyright (C) 2018 CERN.
+
+    Asclepias Broker is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+.. include:: ../README.rst
+
+User's Guide
+------------
+
+This part of the documentation will show you how to get started in using
+Asclepias Broker.
+
+.. toctree::
+   :maxdepth: 2
+
+   installation
+   configuration
+   usage
+
+API Reference
+-------------
+
+If you are looking for information on a specific function, class or method,
+this part of the documentation is for you.
+
+.. toctree::
+   :maxdepth: 2
+
+   api
```

Comment:
In addition to traditional API docs, it would be really helpful to have a page that gives a 10,000ft view of the code - that is, what are the main components of the broker, and how do they fit together? Kind of like the start of this page, with the diagram: http://docs.glueviz.org/en/stable/python_guide/data_tutorial.html